src/wizards/tokenSwapDealWizard/stages/leadDetailsStage/leadDetailsStage.html

```diff
@@ -47,7 +47,7 @@ <h2 data-test="section-title" class="sidebarHeader heading heading3 title">Propo
     <div class="stageSection" if.bind="isOpenProposalWizard">
       <div class="stageSectionSidebar">
         <h2 class="sidebarHeader heading heading3 title">Keep Admin Rights?</h2>
-        <p class="sidebarText">Toggle this option on if you would like to retain the Proposal Lead role to yourself. This will allow you to steer all the offers incoming to your Open Proposal.</p>
+        <p class="sidebarText">Toggle this option on if you would like to retain the Proposal Lead role to yourself. This will allow you to remain admin and manage all the offers incoming to your Open Proposal.</p>
```

Comment:
Should be "remain Proposal Lead and manage"

Reply:
What should we do with the word "Admin" in the section title?
"Keep Admin Rights?"

Reply:
Should be "Keep the Project Lead Role", or at minimum should be "Administration".  We should not be using abbreviations (with respect to non-native english speakers)